http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -195,6 +196,31 @@ protected OidcClientConfiguration internalBuild(final OidcJsonConfiguration oidc
 
         oidcClientConfiguration.setTokenSignatureAlgorithm(oidcJsonConfiguration.getTokenSignatureAlgorithm());
 
+        String tmpLogoutPath = oidcJsonConfiguration.getLogoutPath();
+        log.debugf("property LOGOUT_PATH: " + tmpLogoutPath);
+        if (tmpLogoutPath != null) {
```

Comment:
Just noticed that in the rest of this class, it's common to set the default value in the else statement, e.g.,

```
if (oidcJsonConfiguration.getLogoutPath() != null) {
...
} else {
    oidcClientConfiguration.setLogoutPath(DEFAULT_LOGOUT_PATH)
}
```

Reply:
The same could be done for each of the options below as well.

Reply:
done